### PR TITLE
ACMS-000: Use stable release for all acquia_cms modules.

### PR DESCRIPTION
**Pull request by deepakmishra2**
**Motivation**

Fixes #NNN

**Proposed changes**

Added dev releases of acquia_cms_common module

**Alternatives considered**

**Testing steps**

**Merge requirements**
- [ ] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

File: modules/acquia_cms_audio/composer.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "description": "Provides an Audio media type and related configuration.",
     "license": "GPL-2.0-or-later",
     "require": {
-        "drupal/acquia_cms_common": "1.x-dev || 2.x-dev || 3.x-dev",
+        "drupal/acquia_cms_common": "1.x-dev || 2.x-dev || 3.x-dev || 3.1.x-dev || 3.2.x-dev",
```
Comment (rajeshreeputra) [inline modules/acquia_cms_audio/composer.json:7]:
```suggestion
        "drupal/acquia_cms_common": "1.x-dev || 2.x-dev || 3.x-dev || 3.1.x-dev || 3.2.x-dev || 3.x-dev",
```

File: modules/acquia_cms_component/composer.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
     "description": "Provides component and related configuration for Acquia CMS.",
     "license": "GPL-2.0-or-later",
     "require": {
-        "drupal/acquia_cms_common": "1.x-dev || 2.x-dev || 3.x-dev",
+        "drupal/acquia_cms_common": "1.x-dev || 2.x-dev || 3.x-dev || 3.1.x-dev || 3.2.x-dev",
```
Comment (rajeshreeputra) [inline modules/acquia_cms_component/composer.json:7]:
```suggestion
        "drupal/acquia_cms_common": "1.x-dev || 2.x-dev || 3.x-dev || 3.1.x-dev || 3.2.x-dev || 3.x-dev",
```

**Review by rajeshreeputra — CHANGES_REQUESTED** [review]

File: modules/acquia_cms_audio/composer.json (hunk 1 above)
Comment (chandan-singh7929) [reply modules/acquia_cms_audio/composer.json:7]: why we are adding `3.x-dev` twice?

**Review by chandan-singh7929 — APPROVED** [review]
Remove duplicate `3.x-dev` from all the changes, rest looks good

File: modules/acquia_cms_audio/composer.json (hunk 1 above)
Comment (deepakmishra2) [reply modules/acquia_cms_audio/composer.json:7]: yes its already there

**Comment by deepakmishra2** [comment]
> Remove duplicate `3.x-dev` from all the changes, rest looks good

3.x-dev is added only once , suggested changes by pravin shows its should be added twice

**Review by rajeshreeputra — APPROVED** [review]

**Review by apathak18 — APPROVED** [review]

**Review by vishalkhode1 — APPROVED** [review]

**Pull request merged**